include/cvc5/cvc5_proof_rule.h

```diff
@@ -1293,13 +1293,15 @@ enum ENUM(ProofRule) : uint32_t
    *
    * .. math::
    *
-   *   \inferrule{\forall x_1\dots x_n.\> F\mid t_1,\dots,t_n, (id\, (t)?)?}
+   *   \inferrule{\forall x_1\dots x_n.\> F\mid (t_1,\dots,t_n), (id\, (t)?)?}
```

Comment:
```suggestion
   *   \inferrule{\forall x_1\dots x_n.\> F\mid (t_1 \dots t_n), (id\, (t)?)?}
```
should update for s-expression also, right?